docs/changelog/118559.yaml

```diff
@@ -0,0 +1,11 @@
+pr: 118559
+summary: Make logsdb general available
+area: Logs
+type: feature
+issues: []
+highlight:
+  title: Make logsdb general available
+  body: >-
+    Logsdb has been GA-ed. Logsdb is a feature that allows Elasticsearch to store logs more efficiently.
```

Comment:
suggested body: Logsdb index mode has been GA-ed. Logsdb mode enables Elasticsearch to store logs more efficiently. Logsdb can reduce storage usage up to ~3 times compared to storing logs in Elasticsearch without Logsdb.